AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java

```diff
@@ -507,6 +508,11 @@ private void saveLastDeckId(Long id) {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
+        // Saves the present Night Mode Preference on activity start.
+        isDarkModeAtStart=AnkiDroidApp.getSharedPrefs(getApplicationContext())
```

Comment:
Unsure if we can call it this early. We should move the code below `super.onCreate(savedInstanceState);`
```suggestion
        isDarkModeAtStart = AnkiDroidApp.getSharedPrefs(this)
```